packages/ui-komponenter/src/KopierbarTekst.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactElement, useState } from 'react';
+
+import { Tooltip } from '@navikt/ds-react';
+
+import { createIntl } from '@navikt/ft-utils';
+
+import messages from '../i18n/nb_NO.json';
+
+const intl = createIntl(messages);
+
+type Props = { tekst: string | undefined; children?: ReactElement | string };
+
+export const KopierbarTekst = ({ tekst, children }: Props) => {
+  const kopierTekst = intl.formatMessage({ id: 'KopierbarTekst.Kopier' });
+  const kopiertTekst = intl.formatMessage({ id: 'KopierbarTekst.Kopiert' });
+
+  const [tooltipText, setTooltipText] = useState(kopierTekst);
```

Comment:
For meg hadde det nok vore lettare å lesa dette om state hadde vore ein bolsk variabel. Det er vel kun to variasjonar av state her ([erKopiert, setErKopiert] = useState(false)). Da hadde ein ikkje hatt behov for å følga teksten definert på toppen gjennom flyten.

Reply:
Endret til boolsk state nå!